### PR TITLE
Minor typos in code comments.

### PR DESCRIPTION
**Pull request by tmcgilchrist**

### CODE REVIEW

**Review by coot — CHANGES_REQUESTED** [review]

File: ouroboros-network-framework/src/Ouroboros/Network/Snocket.hs [hunk 1]
```diff
@@ -13,21 +13,21 @@ module Ouroboros.Network.Snocket
...
   , LocalSocket (..)
-  , LocalAddress (..)
-  , localAddressFromPath
   , TestAddress (..)
-  , FileDescriptor (..)
-  , socketFileDescriptor
+  , localAddressFromPath
+  , localSnocket
   , localSocketFileDescriptor
+  , socketFileDescriptor
```
[the remark this answers is not in the document]
Comment (tmcgilchrist) [reply ouroboros-network-framework/src/Ouroboros/Network/Snocket.hs]: Reverted too.

**Review by coot — APPROVED** [review]
LGTM

**Comment by coot** [comment]
bors merge

**Comment by coot** [comment]
bors merge

**Pull request merged**